packages/@aws-cdk/cloudformation-diff/lib/format.ts

```diff
@@ -239,6 +246,10 @@ export class Formatter {
     }
   }
 
+  private formatMove(move?: Move): string {
+    return !move ? '' : chalk.gray('(OR', chalk.italic(chalk.bold('move')), `${move.direction} ${move.stackName}.${move.resourceLogicalId} via refactoring)`);
```

Comment:
Can we make this slightly more visible than grey? Maybe yellow or orange or something?